File: TestFire/setup/TestFireSetup.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TechTalk.SpecFlow;
using OpenQA.Selenium;
using OpenQA.Selenium.Chrome;


namespace TestFire.setup
{
   public class TestFireSetup
    {
        public IWebDriver Driver;

        [BeforeScenario]

        public void BeforeScenario()
        {
            Driver = new ChromeDriver();
            Driver.Manage().Window.Maximize();
        }

        [AfterScenario]

        public void AfterScenario()
        {
            Driver.Close();
            Driver.Quit();

        }
    }
}

File: TestFire/setup/TestFireSetup.ts
import{WebDriver,Capabilities} from 'selenium-webdriver'
import {ChromeDriver} from 'chromedriver'


export class TestFireSetup {
    
    Driver: WebDriver;

    //let {setDefaultTimeout, After, Before, AfetrAll, BeforeAll} = require('cucumber')
    
    @BeforeScenario()
    BeforeScenario() {
        this.Driver = new ChromeDriver();
        this.Driver.manage().window().maximize();
    }
    
    @AfterScenario()
    AfterScenario() {
        this.Driver.close();
        this.Driver.quit();
    }
}